index.php

```diff
@@ -34,7 +34,10 @@
                     </h2>
                     <p><?php echo $clanek["perex"]; ?></p>
                     <p>
-                        <a href="mailto:#"><?php echo $autoriJmena["ondra"]; ?></a> |
+                        <?php foreach ($clanek["autori"] as $autor) { ?>
+                            <?php echo $autor;?>
```

Comment:
neni tohle lepsi bez ukonceni php jen s 
```php
{
   echo $autor;
}
```
?

Reply:
Taky by se mi asi vic libilo, kdyby se ten foreach sfouknul v ramci jednoho php kodu (jakoze `<?php ... ?>`)

Reply:
Jenomže:
- takhle je to použité i u toho vnějšího foreache
- v dalším cvičení  se tam přidává `<a href="mailto...">`, a abys to napsal, budeš muset buďto escapovat (to jsme je neučily) nebo psát apostrofy (taky jsme jim neříkali), nebo se upíšeš přes tečky.